_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+    }
+
+    .xl-col-12 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(full); };
+    }
+
+    // shifts - pre
+    .xl-pre-1 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(1); };
+    }
+
+    .xl-pre-2 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(2); };
+    }
+
+    .xl-pre-3 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(3); };
+    }
+
+    .xl-pre-4 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(4); };
+    }
+
+    .xl-pre-5 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(5); };
+    }
+
+    .xl-pre-6 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(6); };
+    }
+
+    .xl-pre-7 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(7); };
+    }
+
+    .xl-pre-8 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(8); };
+    }
+
+    .xl-pre-9 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(9); };
+    }
+
+    .xl-pre-10 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(10); };
+    }
+
+    .xl-pre-11 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include pre(11); };
+    }
+
+    // shifts - post
+    .xl-post-1 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(1); };
+    }
+
+    .xl-post-2 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(2); };
+    }
+
+    .xl-post-3 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(3); };
+    }
+
+    .xl-post-4 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(4); };
+    }
+
+    .xl-post-5 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(5); };
+    }
+
+    .xl-post-6 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(6); };
+    }
+
+    .xl-post-7 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(7); };
+    }
+
+    .xl-post-8 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(8); };
+    }
+
+    .xlxl-post-9 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(9); };
+    }
+
+    .xl-post-10 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(10); };
+    }
+
+    .xl-post-11 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include post(11); };
+    }
+}
+
+// ----------------------------
+// #GALLERY
+// ----------------------------
+.gallery {
+    @extend %grid-gallery;
+}
+
+// ----------------------------
+// #GALLERY - CANNED
+// ----------------------------
+.grid-manual {
+
+    .gallery {
+        @include layout($susy-default, clean);
```

Comment:
NOTE: passing the ``$susy-default`` layout settings in here since galleries can handle margins and nth-children well.